contrib_bots/lib/CommuteBot/doc.md

```diff
@@ -0,0 +1,31 @@
+This bot will allow briefings of estimated travel times, distances and fare information for transit travel.
```

Comment:
You have screenshots you are not using in the docs - they might be helpful for the users.

As for the screenshot files:
- I can't load the files themselves, so I'm not sure if they are helpful, could you check if they're uploaded correctly?
- Make sure there are no whitespaces in file names, use a dash (`-`).
- I don't think `trump to white house` is appropriate here, we don't want to antagonize any users - could you remove any references of that?